### ISSUE TITLE
Not able to download to particular version

### ISSUE THREAD

**Issue by krishna3008** [SergeyPirogov/webdriver_manager#272]
I manually giving the suitable version as a input to download the driver but still i am seeing the same error.
My Code is:

```
import os, time
from selenium import webdriver
from selenium.webdriver.common.by import By
from webdriver_manager.microsoft import EdgeChromiumDriverManager
from selenium.webdriver.edge.options import Options
from selenium.webdriver.edge.service import Service

chrome_options = Options()
prefs = {'safebrowsing.enabled': 'false','download.default_directory' : os.getcwd() + os.path.sep}
chrome_options.add_experimental_option("prefs", prefs)
chrome_options.add_experimental_option("useAutomationExtension", False)
chrome_options.add_argument('headless')

driver = webdriver.Edge(service = Service(EdgeChromiumDriverManager(edgevesion()).install()), options = chrome_options)

driver.get (url)
```

Here `edgevesion()` will give the browser version as output

I am using the version 3.5.2

I got the error as

```
['94.0.992.38']
Done
Traceback (most recent call last):

  File "C:\Users\AppData\Local\Temp/ipykernel_40836/3212575664.py", line 1, in <module>
    runfile('C:/Users/Desktop/Remote_work/EdgeTest/Edge.py', wdir='C:/Users/Desktop/Remote_work/EdgeTest')

  File "C:\Users\Anaconda3\lib\site-packages\debugpy\_vendored\pydevd\_pydev_bundle\pydev_umd.py", line 167, in runfile
    execfile(filename, namespace)

  File "C:\Users\Anaconda3\lib\site-packages\debugpy\_vendored\pydevd\_pydev_imps\_pydev_execfile.py", line 25, in execfile
    exec(compile(contents + "\n", file, 'exec'), glob, loc)

  File "C:/Users/Edge.py", line 47, in <module>
    driver = webdriver.Edge(service = Service(EdgeChromiumDriverManager(version=edgevesion()).install()), options = chrome_options)

  File "C:\Users\Anaconda3\lib\site-packages\webdriver_manager\microsoft.py", line 50, in __init__
    super().__init__(path, log_level, print_first_line, cache_valid_range)

  File "C:\Users\Anaconda3\lib\site-packages\webdriver_manager\manager.py", line 13, in __init__
    log("====== WebDriver manager ======", level=log_level)

  File "C:\Users\Anaconda3\lib\site-packages\webdriver_manager\logger.py", line 25, in log
    _init_logger(level, name, first_line, formatter)

  File "C:\Users\Anaconda3\lib\site-packages\webdriver_manager\logger.py", line 19, in _init_logger
    _logger.setLevel(level)

  File "C:\Users\Anaconda3\lib\logging\__init__.py", line 1421, in setLevel
    self.level = _checkLevel(level)

  File "C:\Users\Anaconda3\lib\logging\__init__.py", line 201, in _checkLevel
    raise TypeError("Level not an integer or a valid string: %r" % level)

TypeError: Level not an integer or a valid string: None
```

**Comment by aleksandr-kotlyar** [comment]
Should be resolved by this commit https://github.com/SergeyPirogov/webdriver_manager/commit/1b135b5f1762c4a00db094427bf586b1c83ffe0e

Will be available in webdriver-manager 3.5.3.

**Comment by aleksandr-kotlyar** [comment]
@krishna3008  Thank you for your report! Please try [3.5.3](https://pypi.org/project/webdriver-manager/3.5.3/) with a fix! Should work now.
